js/index.js

```diff
@@ -1 +1,113 @@
 // Your code goes here
+//Fun Bus text transformed when scrolling up and down
+
+const titleH1 = document.querySelector('h1');
+
+window.addEventListener('wheel', e => {
+  if(e.wheelDelta === -315) {
+    titleH1.textContent = 'Wheeeeelssss';
+  } else {
+    titleH1.textContent = 'Uppppppppppp';
+  }
+});
+
+//Nav items random colors
+
+const navItems = document.querySelectorAll('a');
```

Comment:
This was so smart and I love the random color effect it created

Reply:
Thank you! :)